ModularTegustation/tegu_items/gadgets/unpowered.dm

```diff
@@ -100,30 +104,25 @@
 /obj/item/commandprojector/afterattack(atom/target, mob/user, proximity_flag)
 	. = ..()
 	if(cooldown <= world.time)
-		for(var/obj/effect/temp_visual/V in range(get_turf(target), 0))
-			if(is_type_in_typecache(V, commandtypes))
-				qdel(V)
-				return
-		switch(commandtype)
-			if(1)
-				new /obj/effect/temp_visual/commandMove(get_turf(target))
-			if(2)
-				new /obj/effect/temp_visual/commandWarn(get_turf(target))
-			if(3)
-				new /obj/effect/temp_visual/commandGaurd(get_turf(target))
-			if(4)
-				new /obj/effect/temp_visual/commandHeal(get_turf(target))
-			if(5)
-				new /obj/effect/temp_visual/commandFightA(get_turf(target))
-			if(6)
-				new /obj/effect/temp_visual/commandFightB(get_turf(target))
-			else
-				to_chat(user, "<span class='warning'>CALIBRATION ERROR.</span>")
+		for(var/obj/effect/temp_visual/HoloCommand/V in get_turf(target))
+			qdel(V)
+			return
```

Comment:
Yes that is intended since its the "delete command" function where you can remove a command. If i let it continue it would then go on click spam cooldown which i dont think would happen unless your rapidly moving your mouse to several holo commands..